src/cli/onefuzz/status/cache.py

```diff
@@ -271,10 +290,31 @@ def add_job(self, job_id: UUID, state: JobState, job: Optional[Job] = None) -> N
             except Exception:
                 logging.debug("unable to find job: %s", job_id)
 
+    def should_render_job(self, job: Job) -> bool:
+        if self.job_filters.job_id is not None:
+            if job.job_id not in self.job_filters.job_id:
+                logging.info("skipping:%s", job)
```

Comment:
Should this maybe be at a higher verbosity level? We'd only want to see this if debugging, right?

Reply:
The logs are not actually printed here.  the TUI hides the logging output